src/aklite_client_ext.cc

```diff
@@ -123,7 +123,7 @@ GetTargetToInstallResult AkliteClientExt::GetTargetToInstall(const CheckInResult
     auto apps_to_update = client_->appsToUpdate(Target::fromTufTarget(current));
     if (force_apps_sync || !apps_to_update.empty()) {
       // Force installation of apps
-      res.selected_target = current;
+      res.selected_target = checkin_res.SelectTarget(current.Version());
```

Comment:
Will it work in the offline case? The checkin/TUF DB can have a target json with a list of all apps, while the target json stored for current target just some of the apps, and the offline update bundle includes only the shortlisted apps?

IIUC, we need addition `if` statement, `if (is_offline_more)`?

Reply:
Ok, makes sense.